spec/Appendix B -- Grammar Summary.md

```diff
@@ -172,3 +173,57 @@ NonNullType :
 Directives : Directive+
 
 Directive : @ Name Arguments?
+
+TypeSystemDefinition :
+  - SchemaDefinition
+  - TypeDefinition
+  - TypeExtensionDefinition
+  - DirectiveDefinition
+
+SchemaDefinition : schema Directives? { OperationTypeDefinition+ }
+
+OperationTypeDefinition : OperationType : NamedType
+
+TypeDefinition :
+  - ScalarTypeDefinition
+  - ObjectTypeDefinition
+  - InterfaceTypeDefinition
+  - UnionTypeDefinition
+  - EnumTypeDefinition
+  - InputObjectTypeDefinition
+
+ScalarTypeDefinition : scalar Name Directives?
+
+ObjectTypeDefinition : type Name ImplementsInterfaces? Directives? { FieldDefinition+ }
```

Comment:
Also `graphql-js` expects an empty one to work: https://github.com/graphql/graphql-js/blob/master/src/language/__tests__/schema-parser-test.js#L168

```
const body = 'type Hello implements World { }';
```